lib/if.c

```diff
@@ -1423,6 +1423,8 @@ static int lib_interface_create(enum nb_event event,
 #else
 		ifp = if_get_by_name(ifname, vrf->vrf_id);
 #endif /* SUNOS_5 */
+
+		ifp->configured = true;
 		nb_running_set_entry(dnode, ifp);
```

Comment:
I like that pull request.
it is clever than checking the whole interface configuration (for instance look that daemon parameters are different from default, or ask to yang_dnode_get..).

just to be sure, this handler ( lib_interface_create() ) is called  only upon configuration, and not upon interface presence on the system too ?

Reply:
that is my understanding as well.